File: EJ1-Components-exmples/UploadBox/WebForms/UploadBox/removeFile.ashx.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Web;

namespace UploadBox
{
    /// <summary>
    /// Summary description for removeFile
    /// </summary>
    public class removeFile : IHttpHandler
    {

        public void ProcessRequest(HttpContext context)
        {
            System.Collections.Specialized.NameValueCollection s = context.Request.Params;
            string fileName = s["fileNames"];
            string targetFolder = HttpContext.Current.Server.MapPath("uploadfiles");
            var customdata = context.Request.Params[0];
            if (Directory.Exists(targetFolder))
            {
                string physicalPath = targetFolder + "\\" + fileName;
                if (System.IO.File.Exists(physicalPath))
                {
                    System.IO.File.Delete(physicalPath);
                }
            }
            context.Response.ContentType = "text/plain";
            context.Response.Write("Hello World");
        }
        public bool IsReusable
        {
            get
            {
                return false;
            }
        }
    }
}

File: EJ1-Components-exmples/UploadBox/WebForms/UploadBoxVB_Save_Remove/removeFile.ashx.vb
Imports System.IO
Imports System.Web
Imports System.Web.Services

Public Class removeFile
    Implements System.Web.IHttpHandler

    Sub ProcessRequest(ByVal context As HttpContext) Implements IHttpHandler.ProcessRequest

        Dim s As System.Collections.Specialized.NameValueCollection = context.Request.Params
        Dim fileName As String = s("fileNames")
        Dim targetFolder As String = HttpContext.Current.Server.MapPath("uploadfiles")
        If Not Directory.Exists(targetFolder) Then
            Directory.CreateDirectory(targetFolder)
        End If
        Dim physicalPath As String = targetFolder + "\" + fileName
        If System.IO.File.Exists(physicalPath) Then
            System.IO.File.Delete(physicalPath)
        End If

    End Sub

    ReadOnly Property IsReusable() As Boolean Implements IHttpHandler.IsReusable
        Get
            Return False
        End Get
    End Property

End Class